jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java

```diff
@@ -568,13 +568,17 @@ protected void onClientRequestFailure(HttpServletRequest clientRequest, Request
         boolean aborted = proxyRequest.abort(failure);
         if (!aborted)
         {
-            int status = failure instanceof TimeoutException ?
-                    HttpStatus.REQUEST_TIMEOUT_408 :
-                    HttpStatus.INTERNAL_SERVER_ERROR_500;
+            int status = clientRequestStatus(failure);
             sendProxyResponseError(clientRequest, proxyResponse, status);
         }
     }
 
+	protected int clientRequestStatus(Throwable failure) {
+		return failure instanceof TimeoutException ?
```

Comment:
Too much indentation for `return`. Use 4 spaces, no tabs.
Broken line gets indented 8 spaces.